pyflask/manageNeuroconv/manage_neuroconv.py

```diff
@@ -71,15 +80,29 @@ def convert_to_nwb(info: dict) -> bool:
     Function used to convert the source data to NWB format using the specified metadata.
     """
 
+    nwbfile_path = info["nwbfile_path"]
+
+    runStubTest = ("stub_test" in info) and info["stub_test"]
+
+    # add a subdirectory to a filepath if stub_test is true
+    if runStubTest:
+        filename = os.path.basename(nwbfile_path)
+        dirname = os.path.join(os.path.dirname(nwbfile_path), ".stubs")
+        nwbfile_path = os.path.join(dirname, filename)
+        check_dir(dirname)
```

Comment:
```suggestion
    run_stub_test = ("stub_test" in info) and info["stub_test"]

    # add a subdirectory to a filepath if stub_test is true
    if run_stub_test:
        filename = os.path.basename(nwbfile_path)
        dirname = os.path.join(os.path.dirname(nwbfile_path), ".stubs")
        nwbfile_path = os.path.join(dirname, filename)
        check_dir(dirname)
```
Always snake case for instances of classes and variables